x-pack/solutions/search/plugins/enterprise_search/server/deprecations/index.ts

```diff
@@ -0,0 +1,366 @@
+ */
+export function getEnterpriseSearchNodeDeprecation(
+  config: ConfigType,
+  isCloud: boolean
+): DeprecationsDetails[] {
+  if (config.host) {
+    const steps = [];
+    if (isCloud) {
+      steps.push(
+        ...[
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.gotocloud', {
+            defaultMessage: 'Go to cloud.elastic.co',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.clickedit', {
+            defaultMessage: "Click the 'Edit' tab",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.scrolldown', {
+            defaultMessage: "Scroll down to the 'Enterprise Search' section",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.clickx', {
+            defaultMessage: "Click the red 'X' by 'Enterprise Search instances'",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.nocapacity', {
+            defaultMessage: 'You should no longer see any Enterprise Search capacity',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.clicksave', {
+            defaultMessage: "Click 'Save' and confirm",
+          }),
+        ]
+      );
+    } else {
+      steps.push(
+        ...[
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.stopent', {
+            defaultMessage: 'Stop all your Enterprise Search nodes',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.stopkibana', {
+            defaultMessage: 'Stop all your Kibana nodes',
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.removeconfig', {
+            defaultMessage: "Edit 'kibana.yml' to remove 'enterpriseSearch.host'",
+          }),
+          i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.restart', {
+            defaultMessage: 'Restart Kibana',
+          }),
+        ]
+      );
+    }
+    return [
+      {
+        level: 'critical',
+        deprecationType: 'feature',
+        title: i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.title', {
+          defaultMessage: 'Enterprise Search host(s) must be removed',
+        }),
+        message: i18n.translate('xpack.enterpriseSearch.deprecations.entsearchhost.message', {
+          defaultMessage:
+            'Enterprise Search is not supported in versions >= 9.x. ' +
+            'You must remove Enterprise Search node(s) from your deployment to proceed with the upgrade. ' +
+            'Note that once Enterprise Search is stopped, products such as App Search, Workplace Search, and Elastic Crawler ' +
+            'will cease to operate. Native Connectors will also stop running syncs, between when Enterprise Search is stopped ' +
+            'and when the 9.x upgrade completes. For full details, see the documentation.',
```

Comment:
```suggestion
            'will cease to operate. Native Connectors will also stop running syncs until the 9.x upgrade completes. ' +
            '. For full details, see the documentation.',
```

Think it's clear from previous sentence that the node is stopped, so this one can be briefer.

Reply:
(copyedit, caveat not sure if this will render nicely)